src/Task_1/DoWhile.java

```diff
@@ -0,0 +1,51 @@
+package Task_1;
+
+import java.util.Scanner;
+
+public class DoWhile {
+
+    public static void main(String args[]){
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter number: ");
+        int n = reader.nextInt();
+        int i = 0;
+
+        if(n > 0){
+
+            do{
+
+                if(i%2==0){
+
+                    System.out.println(i + " even");
+                }
+
+                else{
+
+                    System.out.println(i + " not even");
+                }
+                i++;
+            }
+            while (i <= n);
+        }
+
+        if (n < 0){
```

Comment:
You could use an 'else if' statement here. 
This way, if n > 0 is true you avoid this unnecessary check here, since n can't be  both greater and less than 0.
Also what's the application behavior if your input is 0?